README.md

```diff
@@ -63,8 +63,19 @@ $ vcpkg install openssl
 $ vcpkg install rapidjson
 $ vcpkg install protobuf protobuf[zlib]
 $ vcpkg install spdlog 
-$ make build
-$ make test
+```
+
+#### Setup SDK local repository
+```
+$ git clone https://github.com/eclipse-uprotocol/uprotocol-sdk-cpp.git
+$ /usr/bin/cmake --no-warn-unused-cli -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_BUILD_TYPE:STRING=Debug -DCMAKE_C_COMPILER:FILEPATH=/usr/bin/c89-gcc -S uprotocol-sdk-cpp -B uprotocol-sdk-cpp/build -G Ninja
+```
```

Comment:
I would put a note in that if they use VSCode with the CMake & CMake Tools plugin, this step will be done automatically. If they do not have a build directory autogenerated, then they need to manually run cmake with the example below.